solutions/LC matrix-block-sum.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: lc-MatrixBlockSum
+source: LeetCode
+title: Matrix Block Sum
+author: Jesse Choe
+---
+
+**Time Complexity:** $\mathcal O(NM)$
+
+<LanguageSection>
+
+<CPPSection>
+
+## Implementation
+
+```cpp
+class Solution {
+    public:
+            vector<vector<int>> matrixBlockSum(vector<vector<int>>& mat, int K) {
+                int n = mat.size(), m = mat[0].size();
+                int arr[n][m];
```

Comment:
I remember something about this not being cpp standard -- use 2D vector or static array size.

Reply:
Likewise for dp array below

Reply:
yes this is not cpp standard use vector